recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
```

Comment:
Not when it comes to Fortran. That is the only way we have to build Fortran on OS X. We will need to make sure the `clang` stuff stays entact. We don't want this to start shipping `libgcc` on OS X if we can avoid it.